engine/utils/jinja_utils.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <json/value.h>
+#include <string>
+
+#include "extensions/remote-engine/template_renderer.h"
+#include "utils/chat-template.hpp"
+#include "utils/result.hpp"
+
+namespace jinja {
```

Comment:
I think we use `inja` not `jinja`

Reply:
I think jinja is more accurate in semantic. inja is a library tool.